frontend/src/components/HomePage/HomePage.scss

```diff
@@ -1,16 +1,44 @@
 .homePageElements {
   display: flex;
+  flex-direction: row;
```

Comment:
already in row when using flex by default

Reply:
Corrected